app/ide-desktop/client/tests/gettingStarted.spec.ts

```diff
@@ -0,0 +1,164 @@
+/** @file A series of tests designed for testing 'Getting Started with Enso Analytics'. */
+
+import { expect } from 'playwright/test'
+import {
+  closeWelcome,
+  createNewComponent,
+  createNewProject,
+  loginAsTestUser,
+  test,
+  visualizeData,
+} from './electronTest'
+
+// First excercise in Enso Analytics 101
+test('Exercise 1', async ({ page }) => {
+  await loginAsTestUser(page)
+  await closeWelcome(page)
+
+  // ---------------- Objective 1 ----------------
+  await test.step('Objective 1: Let’s read a single sheet in from Excel', async () => {
+    await createNewProject(page)
+
+    const addComponent = page.getByLabel('Add Component (Enter)')
+    await expect(addComponent).toBeVisible()
+    await addComponent.click()
+
+    const dataReadEntry = page.locator('.ComponentEntry', { hasText: /^Data\.read$/ })
+    await expect(dataReadEntry).toBeVisible()
+    await dataReadEntry.click()
+
+    const urlBox = page.getByTestId('widget-text-content')
+    await expect(urlBox).toBeVisible()
+    await urlBox.fill('Samples/Data/sample_bank_data.xlsx')
+
+    await Promise.race([
+      page.getByLabel('Show visualization (Space)').waitFor({ state: 'visible', timeout: 5000 }),
+      page
+        .getByText(/file not found/i)
+        .waitFor({ state: 'visible', timeout: 5000 })
+        .catch(() => null),
+    ])
+
+    await visualizeData(page)
+
+    // Choosing the first sheet
+    await expect(page.getByText('Sheet1')).toBeVisible()
+    await page.getByText('Sheet1').dblclick()
+
+    await visualizeData(page)
+  })
+
+  // ---------------- Objective 2 ----------------
+  await test.step('Objective 1: Filter Data to find “exception” records', async () => {
+    // Adding set component
+    await createNewComponent(page)
+    await page.locator('.ComponentEntry', { hasText: 'set' }).click()
+
+    // Set parameters
+    await page.locator('.WidgetSelection.clickable').filter({ hasText: 'value' }).click()
+    await page.getByRole('button', { name: '<Simple Expression>', exact: true }).click()
+
+    await page.locator('.widgetApplyPadding', { hasText: 'input' }).click()
```

Comment:
Something like `.WidgetTopLevelArgument` should be better than `.widgetApplyPadding`. The latter is a utility class that can disappear or be renamed at any time.